src/apps/loyalty/containers/forgot_password/index.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from "react"
```

Comment:
yeah that's a good point, we can definitely extract the login and forgot password to be reused outside of the loyalty app